pkgs/development/python-modules/zebrafy/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  setuptools-scm,
+  pillow,
+  pypdfium2,
+  pytest-cov,
+}:
+
+buildPythonPackage rec {
+  pname = "zebrafy";
+  version = "1.2.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "miikanissi";
+    repo = "zebrafy";
+    tag = version;
+    hash = "sha256-B8jrFQh5swDMfYjdMcY0Hh2VAzknDwarDKVAML6F2r4=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    setuptools-scm
+    pillow
+    pypdfium2
+  ];
+
+  pythonImportsCheck = [ "zebrafy" ];
+
+  nativeCheckInputs = [
+    pytest-cov
+  ];
+
+  meta = {
+    description = "Python library for converting PDF and images to and from Zebra Programming Language";
+    downloadPage = "https://github.com/miikanissi/zebrafy";
+    changelog = "https://github.com/miikanissi/zebrafy/releases/tag/${version}";
+    homepage = "https://zebrafy.readthedocs.io/en/latest/";
+    license = lib.licenses.lgpl3;
```

Comment:
```suggestion
    license = lib.licenses.lgpl3Plus;
```

Reply:
It isn't Plus as far as I can tell.

Reply:
Check again.

Reply:
Helpful. Where exactly do you see this? https://github.com/miikanissi/zebrafy/blob/master/pyproject.toml#L56